eslint.config.mjs

```diff
@@ -109,6 +120,18 @@ export default [
         }
       ]],
 
+      'no-await-in-loop': 'error',
+      'no-else-return': ['error', { allowElseIf: true }],
```

Comment:
you know i can understand that this else return case is equivalent withotu the else, but what's the problem with doing it anyway ? is there any performance downside ? because sometimes i use it explicitly as a semantic form, to organize the code in a more intuitive way.
This rule sounds like it's not fixing actual pain ?

Reply:
It's for better readability. Less indentation improves understanding of code. Else cases also increase the complexity for readers. The allowElseIf is just active to minimize the changes right now. I plan on changing that later.